### DISCUSSION TITLE
Code lens error for Gradle project having dependent module

### DISCUSSION THREAD

**Discussion by preetipadalia** [getgauge/gauge#1908] [General]
Describe the bug
Code lens are not showing for Gradle project having dependent module.
I have two modules in my project.

Core module
Test module
Test is dependent on Core.
Gauge specs are available in Test module. When i am opening the project in VSCODE the code lens are not showing. The logs shows it did not find the dependencies from the dependent module.

To Reproduce
Steps (or project) to reproduce the behavior:

Create a gradle project
Add two modules
create the gauge project in one module and add the other module dependency
See error
Logs

**dependent library error logs**
10-02-2021 16:43:56.300 [Gauge] [ERROR] Error occurred while waiting for runner process to finish.
Error : exit status 1
10-02-2021 16:43:56.300 [Gauge] [DEBUG] Gauge could not initialize. Install 'vscjava.vscode-java-pack' extension for code insights. For more information see[Problems](command:workbench.actions.view.problems), check logs.[Troubleshooting](https://docs.gauge.org/troubleshooting.html?language=javascript&ide=vscode#gauge-could-not-initialize-for-more-information-see-problems)
Some of the gauge lsp feature will not work as expected.
10-02-2021 16:43:56.300 [Gauge] [INFO] LangServer: reading on stdin, writing on stdout
10-02-2021 16:43:56.305 [Gauge] [ERROR] notif #0: initialized: current runner is not compatible with gauge LSP
Expected behavior
It should work as expected

Versions:

OS [e.g. 10.15.6 (19G2021) please be specific ]
Java version
    openjdk version "11.0.10" 2021-01-19 LTS
    OpenJDK Runtime Environment Corretto-11.0.10.9.1 (build 11.0.10+9-LTS)
    OpenJDK 64-Bit Server VM Corretto-11.0.10.9.1 (build 11.0.10+9-LTS, mixed mode)
Maven/Gradle version
------------------------------------------------------------
 Gradle 6.8.2
------------------------------------------------------------

Build time:   2021-02-05 12:53:00 UTC
Revision:     b9bd4a5c6026ac52f690eaf2829ee26563cad426

Kotlin:       1.4.20
Groovy:       2.5.12
Ant:          Apache Ant(TM) version 1.10.9 compiled on September 27 2020
JVM:          11.0.10 (Amazon.com Inc. 11.0.10+9-LTS)
OS:           Mac OS X 10.15.7 x86_64
Gauge version: 1.1.7
Commit Hash: 5d86b72

Plugins
-------
html-report (4.0.12)
java (0.7.13)
js (2.3.15)
reportportal (1.11.1)
screenshot (0.0.1)
spectacle (0.1.4)

**Comment by sriv** [comment]
sorry, seeing this late. Were you able to figure this out?
